test/e2e/suite/conformance/certificates/tests.go

```diff
@@ -605,5 +768,119 @@ func (s *Suite) Define() {
 			err = f.Helper().ValidateCertificate(f.Namespace.Name, certName, validation.CertificateSetForUnsupportedFeatureSet(s.UnsupportedFeatures)...)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		s.it(f, "should fail to obtain a certificate for an invalid ACME dns name", func(issuerRef cmmeta.ObjectReference) {
```

Comment:
It is unclear to me why this is specific to ACME and is invalid. The conformance suite tests should ideally be run against more than one issuer, otherwise it might be better to just back to the issuer specific test suite.

Reply:
In `cert-manager/devel/addon/pebble/chart/templates/configmap.yaml` the google.com DNS Name is blacklisted, so that is where this "invalid ACME DNS Name" comes from.
This test is applicable to both HTTP01 and DNS01 challenges, so therefore it could maybe be useful to still move it to a conformance test?

@JoshVanL please let me know what you think; for now I'll move it back to the issuer specific test suite.

Reply:
I see, understood.

I think having ACME specific tests isn't appropriate here in the conformance suite. Ideally the tests in conformance can be picked up by any issuer implementation, and they can pick and choose which features they support. 

Having a feature which is _only_ for a specific domain blacklist on _this specific_ pebble installation clouds the intention a bit.